test/cmd/builds.sh

```diff
@@ -138,16 +138,23 @@ echo "start-build: ok"
 os::test::junit::declare_suite_end
 
 os::test::junit::declare_suite_start "cmd/builds/cancel-build"
-os::cmd::expect_success_and_text "oc cancel-build ${started} --dump-logs --restart" "Restarted build ${started}."
+os::cmd::expect_success_and_text "oc cancel-build ${started} --dump-logs --restart" "restarted build \"${started}\""
 os::cmd::expect_success 'oc delete all --all'
 os::cmd::expect_success 'oc process -f examples/sample-app/application-template-dockerbuild.json -l build=docker | oc create -f -'
 os::cmd::try_until_success 'oc get build/ruby-sample-build-1'
 # Uses type/name resource syntax to cancel the build and check for proper message
-os::cmd::expect_success_and_text 'oc cancel-build build/ruby-sample-build-1' 'Build ruby-sample-build-1 was cancelled.'
+os::cmd::expect_success_and_text 'oc cancel-build build/ruby-sample-build-1' 'build "ruby-sample-build-1" cancelled'
 # Make sure canceling already cancelled build returns proper message
-os::cmd::try_until_text 'oc cancel-build build/ruby-sample-build-1' 'A cancellation event was already triggered for the build ruby-sample-build-1.'
+os::cmd::try_until_text 'oc cancel-build build/ruby-sample-build-1' 'Build ruby-sample-build-1 is already cancelled'
```

Comment:
This was a really good catch ;-) I realized that when I return error(s) (one or more builds failed to cancel) the return code is always 0...

Which makes me think if we have to return errors in some cases, for example:

`oc cancel-build bc/foo` - foo has 50 builds, some of them are complete, some of them already cancelled...
this command will never exit 0, because we report errors for every single build...
One option would be to suggest to use  `--status=running` to make sure you only cancelling "valid" builds... 
Another option could be to quietly skip the non-cancellable builds... what do you think?

(I think that cancelling an already cancelled build or complete build is not an (exit 1) error, more than a warning or information to user that this command will not do what they think it will do for these builds... so option 2) will work for me, but I would be OK with option 1) as well...).